lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -69,10 +69,16 @@
     description:
       - Enable detached mode to leave the container running in background.
         If disabled, the task will reflect the status of the container run (failed if the command failed).
+    type: bool
+    default: true
   devices:
     description:
       - "List of host device bindings to add to the container. Each binding is a mapping expressed
         in the format: <path_on_host>:<path_in_container>:<cgroup_permissions>"
+  dns_opts:
+    description:
+      - list of DNS options
+    version_added: "2.6"
```

Comment:
This feature existed when the module was created, so `version_added` should not be used. The sanity test will fail due to this, but once that is the only failure we can safely merge. The failure won't occur after the merge.

Reply:
Ok done.